index.bs

```diff
@@ -3194,7 +3194,9 @@ associated with or [=scoped=] to, respectively.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialEntity/name}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that it fits within 64 bytes, if the authenticator stores the value. See [[#sctn-strings-truncation]] about truncation and other considerations.
+        When storing a {{PublicKeyCredentialEntity/name}} member's value,
```

Comment:
Very minor nit: this comma (and in the next diff hunk) read oddly to me, but I'm happy either way.

Reply:
Hm, I see. If you don't mind I'll keep it as is, if only for consistency with the first sentence in the paragraph immediately preceding this (and also the next hunk).